py-polars/Cargo.toml

```diff
@@ -27,7 +27,7 @@ ndarray = { workspace = true }
 num-traits = { workspace = true }
 numpy = { version = "0.21", default-features = false }
 once_cell = { workspace = true }
-pyo3 = { workspace = true, features = ["abi3-py38", "extension-module", "multiple-pymethods", "gil-refs"] }
```

Comment:
"gil-refs" is the transitional feature that suppresses deprecations. There are no deprecations left.